README.md

```diff
@@ -7,4 +7,8 @@ Travis:   [![Travis](https://travis-ci.org/aspnet/Antiforgery.svg?branch=dev)](h
 
 Antiforgery system for generating secure tokens to prevent Cross-Site Request Forgery attacks.
 
-This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+This project is part of ASP.NET Core. You can find documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
+
+Samples can be found in [Entropy](https://github.com/aspnet/Entropy).
```

Comment:
@Eilon - should we rename the repo if it's meant to be a place of Samples? The current name isn't super meaningful in it's current context.